libs/components/forms/testing/src/form-error/form-error-harness.ts

```diff
@@ -3,6 +3,9 @@ import { SkyComponentHarness } from '@skyux/core/testing';
 
 import { SkyFormErrorHarnessFilters } from './form-error-harness.filters';
 
+/**
+ * Harness for interacting with a form error in tests.
```

Comment:
```suggestion
 * Harness to interact with a form error in tests.
```
Super-minor edit for translation/localization prep because gerunds (-ing) can sometimes cause confusion. Feel free to ignore if you prefer the current wording or if you think this changes the context/meaning at all.